webapp/src/features/DashboardProvider/TableDashboard.js

```diff
@@ -0,0 +1,200 @@
+import { DASHBOARD_PROVIDER_MODAL } from '../DashboardProviderModal/dashboardProvider.constants'
+import getDashboardProviderModal from '../DashboardProviderModal/index'
+import Button from 'react-bootstrap/Button'
+import Table from '../../widgets/Table'
+const { saveSelectedVisitId, setSelectedModal } = actions
+
+const TableDashboard = () => {
+	const selectedModal = useSelector(selectModal)
+
+	const dispatch = useDispatch()
+
+	const mockDetails = [
+		{
+			patientName: 'Cody Miles',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'Complete',
+			id: 'aaaa-bbbb-cccc-0001',
+		},
+		{
+			patientName: 'Johnny Walker	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'In Progress',
+			id: 'aaaa-bbbb-cccc-0002',
+		},
+		{
+			patientName: 'Jesse Pinkman	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'In Progress',
+			id: 'aaaa-bbbb-cccc-0003',
+		},
+		{
+			patientName: 'Sarah Daniels	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'Complete',
+			id: 'aaaa-bbbb-cccc-0004',
+		},
+		{
+			patientName: 'Johnny Walker	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'Complete',
+			id: 'aaaa-bbbb-cccc-0005',
+		},
+		{
+			patientName: 'Jesse Pinkman	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'In Progress',
+			id: 'aaaa-bbbb-cccc-0006',
+		},
+		{
+			patientName: 'Sarah Daniels	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'Complete',
+			id: 'aaaa-bbbb-cccc-0007',
+		},
+		{
+			patientName: 'Jesse Pinkman	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'In Progress',
+			id: 'aaaa-bbbb-cccc-0008',
+		},
+		{
+			patientName: 'Jesse Pinkman	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'Complete',
+			id: 'aaaa-bbbb-cccc-0009',
+		},
+		{
+			patientName: 'Johnny Walker	',
+			campus: 'GISD',
+			state: 'Texas',
+			visitReason: 'Headache',
+			language: 'English',
+			totalTime: '00:01:21',
+			status: 'In Progress',
+			id: 'aaaa-bbbb-cccc-0010',
+		},
+	]
+	const statusFormatter = (status) => (
+		<Button
+			variant={status === 'In Progress' ? 'outline-danger' : 'outline-success'}
+			className='f-14 font-weight-bold btn-sm ml-md-auto mr-20'
+		>
+			{status}
+		</Button>
+	)
+
+	const viewFormatter = (id) => {
+		return (
```

Comment:
you can changes the format as you have done for above(without using return statement)